extensions/extensions-plugin.js

```diff
@@ -1,42 +1,46 @@
-var webpack = require('webpack')
-var _ = require('lodash')
-var fs = require('fs')
-var path = require('path')
+const webpack = require('webpack')
+const _ = require('lodash')
+const fs = require('fs')
+const path = require('path')
+const EXTENSIONS_LITE_PATH = path.normalize('/extensions/lite')
+
 
 var { requireExtension } = require('./extensions.js')
 
 const afterResolve = new webpack.NormalModuleReplacementPlugin(/extensions/i, function(res) {
 
   let customEdition = null
+  const userRequest = res.userRequest ? path.normalize(res.userRequest) : res.userRequest
   const [rest, edition] = (res.rawRequest && res.rawRequest.match(/\?edition=(.+)/i)) || []
 
   edition && _.each(['lite', 'pro', 'ultimate'], e => {
     if (edition.toLowerCase().startsWith(e)) {
       customEdition = e
     }
   })
-  
-  if (!res.userRequest 
-    || res.userRequest.indexOf('extensions/empty.jsx') >= 0
-    || (res.userRequest.indexOf('extensions/enterprise') >= 0)) {
+
+  if (!userRequest
+    || userRequest.indexOf(path.normalize('extensions/empty.jsx')) >= 0
+    || (userRequest.indexOf(path.normalize('extensions/enterprise')) >= 0)) {
     return
   }
 
-  const replacement = requireExtension(res.userRequest, customEdition)
-  res.request = res.request.replace('/extensions/lite', replacement)
-  res.resource = res.resource.replace('/extensions/lite', replacement)
+  const replacement = requireExtension(userRequest, customEdition)
+  res.request = res.request.replace(EXTENSIONS_LITE_PATH, replacement)
+  res.resource = res.resource.replace(EXTENSIONS_LITE_PATH, replacement)
 })
 
 const beforeResolve = new webpack.NormalModuleReplacementPlugin(/\+\/views/i, function(res) {
```

Comment:
@slvnperron when is this called?
I want to fix the path issues for Windows, but I'm not sure how to test this?

Reply:
@jsalwen This is called for the Pro version of Botpress. We're always building it internally on OSX so you don't have to worry about it :)